Daniel_Heffner/angular/product_factory/scriptular.js

```diff
@@ -0,0 +1,76 @@
+var appModule = angular.module('product_app', []);
+
+appModule.factory('product_factory', function(){
+	var products = [
+		{name: 'Sample Product', price: 39.95},
+	];
+	var factory = {}
+	factory.getProducts = function(callback){
+		callback(products);
+		return false;
```

Comment:
You should return true here, and reserve returning false for failure conditions. -- or simply always return undefined